cloudevents/adapters/github.md

```diff
@@ -370,6 +550,21 @@ based on the specified event.
 | `time`                | Current time                             |
 | `data`                | Content of HTTP request body             |
 
+### PackageEvent
+
+| CloudEvents Attribute | Value                                  |
+| :-------------------- | :------------------------------------- |
+| `id`                  | "X-GitHub-Delivery" HTTP header value  |
+| `source`              | "repository.url" value                 |
+| `specversion`         | `1.0`                                  |
+| `type`                | `com.github.package.` + "action" value |
+| `datacontentencoding` | Omit                                   |
+| `datacontenttype`     | `application/json`                     |
+| `dataschema`          | Omit                                   |
+| `subject`             | "package.id" value                     |
+| `time`                | "package.(created\|updated)\_at" value |
```

Comment:
it's interesting that the doc says that "created_at" can be null - I wonder how that can be.
I wonder if we should say the value should be taken in this order: updated_at, created_at, current time

Reply:
Yeah GH event can be a bit 'weird' i like the `updated_at|created_at|now()` but a bit painful to write